packit_service/worker/jobs.py

```diff
@@ -216,14 +216,14 @@ def initialize_job_helper(
             "job_config": job_config,
         }
 
-        if isinstance(handler, ProposeDownstreamHandler):
+        if handler_kls == ProposeDownstreamHandler:
```

Comment:
I can see we are already using the equality operator instead of the identity check in different places like [here](https://github.com/packit/packit-service/blob/65c129fd94cf0ace80a19c9ea18e1f67f407bf9c/packit_service/worker/jobs.py#L620).
I think that rewriting this code using the identity check will make it less readable and I think that we have no reason to override the `__eq__` operator in this objects, so probably we should not fear it.

Reply:
You are right, that `in` operator matches using `==`, so yes, you are right, let's keep it the way it is